### ISSUE TITLE
How to select group on data load in grid

### ISSUE THREAD

**Issue by Tsve** [bencripps/react-redux-grid#170]
Hi, im use Grid, map data to state, state loading async. After load im need to set selected items in grid, but i can't get row_ids, have grid some ON_SET_DATA Event for this?

**Comment by bencripps** [comment]
You are right; we don't currently expose an event for when data is loaded into the grid. That seems like a reasonable request, though!

I'm going to mark this as help-wanted, but I will get to this as soon as I can.

Thanks!

**Comment by jiangenhe** [comment]
Looking forward to this feature.
